ADQL.tex

```diff
@@ -2636,7 +2708,7 @@ \subsubsection{UNION}
 
 For a UNION operation to be valid in ADQL, the following criteria MUST be met:
 \begin{itemize}
-    \item the two queries MUST result in the same number of columns
+    \item the two queries MUST result in the same number of columns.
```

Comment:
I am not sure why you have added a full stop to the end of each dot point in this section. Having no full stop seems to be more widely accepted unless each dot point is a sentence starting with a capital letter.

Reply:
It was just because the next item ends with a full stop...it was for consistency. If you tell me it is better without this full stop, I remove it ;-)

Reply:
Done